tests/test_models.py

```diff
@@ -0,0 +1,148 @@
+        donor = DonorFactory()
+        assert str(donor) == donor.full_name
+
+
+class TestCity:
+    def test_unicode(self):
+        city = models.City.objects.create(city='Denton')
+        assert str(city) == city.city
+
+
+class TestCountry:
+    def test_unicode(self):
+        country = models.Country.objects.create(country='United States')
+        assert str(country) == country.country
+
+
+class TestDesigner:
+    def test_unicode(self):
+        designer = models.Designer.objects.create(designer='Shringle')
+        assert str(designer) == designer.designer
+
+
+class TestLabel:
+    def test_unicode(self):
+        label = models.Label.objects.create(label='Chog')
+        assert str(label) == label.label
+
+
+class TestRetailer:
+    def test_unicode(self):
+        retailer = models.Retailer.objects.create(retailer_name='Harrys')
+        assert str(retailer) == retailer.retailer_name
+
+
+class TestRetailerLabel:
+    def test_unicode(self):
+        retailer_label = models.Retailer_Label.objects.create(retailer_label='Rimps')
+        assert str(retailer_label) == retailer_label.retailer_label
+
+
+class TestClassification:
+    def test_unicode(self):
+        classification = models.Classification.objects.create(classification='Style')
+        assert str(classification) == classification.classification
+
+
+class TestLocation:
+    def test_unicode(self):
+        location = models.Location.objects.create(location='B152')
+        assert str(location) == location.location
+
+
+class TestCondition:
+    def test_unicode(self):
+        condition = models.Condition.objects.create(condition='Good')
+        assert str(condition) == condition.condition
+
+
+class TestMaterial:
+    def test_unicode(self):
+        material = models.Material.objects.create(material='Leather')
+        assert str(material) == material.material
+
+
+class TestMeasurement:
+    def test_unicode(self):
+        measurement = models.Measurement.objects.create(measurement='16')
+        assert str(measurement) == measurement.measurement
+
+
+class TestType:
+    def test_unicode(self):
+        object_type = models.Type.objects.create(object_type='Shoes')
+        assert str(object_type) == object_type.object_type
+
+
+class TestPart:
+    def test_unicode(self):
+        part = models.Part.objects.create(part='Left shoe')
+        assert str(part) == part.part
+
+
+class TestObject:
+    def test_show_label(self):
+        object_instance = ObjectFactory()
+        assert object_instance.showLabel() == object_instance.label
+
+    @pytest.mark.xfail(reason="Method doesn't take into account attribute might not exist.")
+    def test_show_label_no_label(self):
+        object_instance = ObjectFactory(lable=None)
+        assert object_instance.showLabel() is None
+
+    def test_show_location(self):
+        object_instance = ObjectFactory()
+        assert object_instance.showLocation() == object_instance.location
+
+    def test_show_designer(self):
```

Comment:
I didn't realize there was more than one of the `show*()` methods. Nevermind my comment then.

We should make an issue to remove them. We can extract that information from the property directly.